anilist.py

```diff
@@ -1,1172 +1,1182 @@
-import collections
-import configparser
-import coloredlogs
-import inflect
-import json
-import logging
-import os
-import re
-import requests
-import sys
-from guessit import guessit
-
-import plexmodule
-
-logger = logging.getLogger('PlexAniSync')
```

Comment:
Again. Entirely removing the `logger` definition is not a  good idea since it allows for easy log filtering. But it could be improved :)